### DISCUSSION TITLE
Hey, I'm getting this error can anyone help me?

### DISCUSSION THREAD

**Discussion by hamsireis** [triplea-game/triplea#9858] [Q&A]
While choosing the map I downloaded, I encountered the following warning: 

Could not parse:jar:file:/C:/Users/Kullanıcı/triplea/downloadedMaps/age_of_tribes-master.zip!/age_of_tribes-master/map/games/Age_of_Tribes_Modern.xml, null  java.lang.NullPointerException

**Comment by RoiEXLab** [comment]
Related to: #9856

Do other maps work? Or is this just happening with this Map, or rather this family of maps (maybe all age-of-tribes versions don't work, but other maps like total war, ww2 classic etc. work)?

**Reply by hamsireis** [reply]
not working

**Reply by hamsireis** [reply]
i tried the other maps and i always getting this error...

**Comment by panther2** [comment]
Closed today. This is an issue rather than a discussion, it lacks information - and it hasn't been updated since 2021.
